setup.py

```diff
@@ -81,6 +82,7 @@
             "netCDF4>=1.5.8",
             "phonopy>=2.4.2",
             "seekpath>=1.9.4",
+            "tblite[ase]>=0.3.0; platform_system=='Linux'",
```

Comment:
I just tried installing `tblite` on macOS and got the error 

```txt
      ../../subprojects/tblite/config/meson.build:46:12: ERROR: Dependency "openmp" not found, tried system
```

I assume that's why `platform_system=='Linux'`. So is Molecule relax currently a Linux-only feature?

Reply:
@janosh: Could you try installing from [conda-forge](https://anaconda.org/conda-forge/tblite)? I don't have a Mac so can't test. Maybe that will give you better luck. I'm not 100% sure how to build it with the ASE support though from Conda.

Reply:
I don't have `conda`. Not hard to install of course but would like to avoid. Thanks for opening the tblite issue though! I don't think lack of Mac support is a deal breaker. We could not make it the default if others disagree. But i think the `Molecule.relax`  will take some time to attract attention anyway and by then the issue you opened might be fixed.

Reply:
Makes sense. I also would be pretty surprised if we found a better default that worked for most of the periodic table with reasonable accuracy. Hard to beat semi-empirical tight binding... until someone trains an M3GNet-equivalent on Molecules data on MP.